core/src/display_object.rs

```diff
@@ -194,12 +194,16 @@ impl BitmapCache {
 #[derive(Clone)]
 pub struct RenderOptions {
     pub apply_transform: bool,
+    pub skip_masks: bool,
```

Comment:
Can you please document these fields for the future? (and the existing one too, if possible)

Especially since `apply_transform` and `apply_matrix` overlap in meaning, at least partially, right?